docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/slurm-known-issues.rst

```diff
@@ -318,6 +318,36 @@ sometimes resolved by additionally installing the ``apptainer-setuid`` package.
       ``slurm.slots_per_node`` to enable multiple CPUs to be used on each node. Without
       ``slurm.slots_per_node`` the job will request ``slots_per_trial`` nodes.
 
+******************
+ PBS Known Issues
+******************
+
+-  The resource pool card on the Cluster tab shows the ``Accelerator`` field as ``unconfigured``.
+   This happens when the GPU type information is not set in the PBS ``resources_available`` object.
+
+   HPC system administrators can set this information using the following set of commands:
+
+   -  Login as a root level user or use ``sudo -i`` to quickly change to a root level user.
```

Comment:
Unclear that this would always be appropriate.   It might be disabled for some systems or against policy.   
In the rest of the docs they just prefix the command examples with 'sudo' to indicate that it needs  privs to run.

Reply:
Using sudo to these commands did not work. After looking up online, the solution was to use `sudo -i` first and then run the commands.